codec-stomp/src/test/java/io/netty/handler/codec/stomp/StompSubframeAggregatorTest.java

```diff
@@ -67,6 +67,43 @@ public void testSingleFrameWithBodyAndContentLength() {
     }
 
     @Test
+    public void testSingleFrameWithBodyAndNoContentLength() {
+
+        ByteBuf incoming = Unpooled.buffer();
+        incoming.writeBytes(StompTestConstants.SEND_FRAME_4.getBytes());
+        channel.writeInbound(incoming);
+
+        StompFrame frame = channel.readInbound();
+        Assert.assertNotNull(frame);
+        Assert.assertEquals(StompCommand.SEND, frame.command());
+        Assert.assertEquals("body", frame.content().toString(CharsetUtil.UTF_8));
+        frame.release();
+
+        Assert.assertNull(channel.readInbound());
+
+    }
+
+    @Test
+    public void testSingleFrameWithSplitBodyAndNoContentLength() {
+
+        for (int n = 0; n < StompTestConstants.SEND_FRAMES_3.length; ++n) {
+            ByteBuf incoming = Unpooled.buffer();
+            incoming.writeBytes(StompTestConstants.SEND_FRAMES_3[n].getBytes());
+            channel.writeInbound(incoming);
+            channel.flush();
+        }
+
+        StompFrame frame = channel.readInbound();
+        Assert.assertNotNull(frame);
+        Assert.assertEquals(StompCommand.SEND, frame.command());
+        Assert.assertEquals("first part of body\nsecond part of body", frame.content().toString(CharsetUtil.UTF_8));
+        frame.release();
+
+        Assert.assertNull(channel.readInbound());
+
```

Comment:
Can I get some help running these checks on my local?  I've tried 'mvn test' and 'mvn validate' and they both pass.  It doesn't seem to catch checkstyle errors in src/test